deploy/charts/external-secrets/templates/rbac.yaml

```diff
@@ -51,6 +51,7 @@ rules:
     - "generators.external-secrets.io"
     resources:
     - "acraccesstokens"
+    - "clustergenerators"
```

Comment:
I added these by hand... any way to do this automatically?

Reply:
we don't do this automatically, it's hand-cranked. We don't use the kubebuilder tooling, IIRC because different components (webhook/cert-controller) need different permissions and IIRC there's no way to get that integrated with kubebuilder tooling.